sklearn/manifold/isomap.py

```diff
@@ -167,9 +167,9 @@ def _fit_transform(self, X):
 
         self.embedding_ = self.kernel_pca_.fit_transform(G)
 
-    @property
```

Comment:
The previous version looks more correct!

Reply:
The description may not be consistent with the code in this case:

https://github.com/scikit-learn/scikit-learn/pull/13635/files

Reply:
When it says "before" it means before the `def`, and not "before the deprecated"